pom.xml

```diff
@@ -5,7 +5,7 @@
 
   <groupId>ch.vorburger</groupId>
   <artifactId>fswatch</artifactId>
-  <version>1.1.1-SNAPSHOT</version>
+  <version>1.1.2-SNAPSHOT</version>
```

Comment:
let's make this the 1.2.0 instead of the 1.1.2, as your contribution here is more than just a minor bug fix, but a new feature